core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/documentation/DocumentationUrls.kt

```diff
@@ -19,6 +19,17 @@ internal interface DocumentationUrls {
         interface RowConditions
     }
 
+    /** [See `NaN` and `NA` on the documentation website.](https://kotlin.github.io/dataframe/nanAndNa.html) */
```

Comment:
Looks like a new pattern to refer on the Documentation site. Is it correct that we did not do it earlier.
It seems good idea, but probably let's discuss it widely.

Because it looks like adding yet one detail to the hierarchy of documents produced by your plugin

Reply:
No, it was used before in Update.kt and Nulls.kt, you can see the other URLs in the file ;)

Reply:
In this way, it looks that we should include somehow the reusing the info in our documentation writing pipeline.
What I mean: imagine, we have a new function or uncovered function, it's great to have an instruction or an example: where you should go and put your sentences. Here, it looks like the main Documentation could be a final judge for our doubts.